### PR TITLE
change screen resolution

### PR DESCRIPTION
**Pull request by Boschio**

### CODE REVIEW

File: javaJframeBE/src/Constants.java
Comment (HusH3535) [inline javaJframeBE/src/Constants.java:1]: tf is this???? nahhh you fire

File: javaJframeBE/src/Main.java
Comment (HusH3535) [inline javaJframeBE/src/Main.java:1]: lgtm

Comment (Boschio) [reply javaJframeBE/src/Main.java:1]: dang

**Review by HusH3535 — APPROVED** [review]
lgtm

**Pull request merged**